pymagicc/io.py

```diff
@@ -936,6 +938,36 @@ class _EmisOutReader(_EmisInReader, _OutReader):
     pass
 
 
+class _InverseEmisReader(_EmisOutReader):
+    _regexp_capture_variable = re.compile(r"(INVERSEEMIS)\.OUT$")
+    def _get_column_headers_and_update_metadata(self, stream, metadata):
+        units = self._read_data_header_line(stream, "UNITS:")
+        variables = convert_magicc7_to_openscm_variables(convert_magicc6_to_magicc7_variables(
+            self._read_data_header_line(stream, "YEARS:")
+        ))
+
+        column_headers = {
+            "variable": variables,
+            "todo": [self._default_todo_fill_value] * len(variables),
+            "unit": units,
+            "region": ["World"] * len(variables),
+        }
+
+        for k in ["unit", "units", "gas"]:
```

Comment:
What's happening here?

Reply:
you need to get rid of units from metadata as it's in the dataframe (having it twice is confusing).